bundles/binding/org.openhab.binding.isy/build.properties

```diff
@@ -0,0 +1,7 @@
+source.. = src/main/java/,\
+           src/main/resources/
+bin.includes = META-INF/,\
+               .,\
+               OSGI-INF/,\
+               lib/isy_inst.jar
```

Comment:
What are the license terms for isy_inst.jar that allow it to be included in openHAB?  Could you add an about.html file in the binding's top directory that explain the origin and license of this library (as some other bindings have done)?